PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.scss

```diff
@@ -1,4 +1,4 @@
-.carousel{
+.carousel1{
```

Comment:
Same redeclaration as last time

Reply:
> Changing classes' names is a 'cute' workaround for no encapsulation bug, but is against every clean code guideline. Please consider angular encapsulation.
> No changes in positioning since last change request.

I think changing names is enough but positioning in css code deffinitely should be fixed.
Also ".carousel1" isn't the best name for class